ui/search.html

```diff
@@ -23,79 +23,111 @@
-                        </div>
+        <main role="main">
+
+            <div class="oae-main-action">
+                <form action="." id="search-form" class="form-search form-horizontal" role="search" aria-label="__MSG__SEARCH__">
+                    <div class="form-group">
+                        <input type="text" id="search-query" class="form-control search-query form-control pull-left" title="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__" autofocus />
+                        <button type="submit" id="search-search-icon" class="btn btn-link pull-left" title="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__">
+                            <i class="fa fa-search"><span class="sr-only">__MSG__SEARCH__</span></i>
+                        </button>
                     </div>
-                </div>
+                </form>
             </div>
 
-            <!-- CONTENT -->
-            <div>
-                <div id="search-content-container" class="row">
-                    <div class="oae-lhnavigation">
-                        <ul id="search-refine-type" class="nav nav-list">
-                            <li class="nav-header">__MSG__REFINE_SEARCH_BY_TYPE_COLON__</li>
-                            <li>
-                                <label class="checkbox"><input type="checkbox" data-type="user" />__MSG__PEOPLE__</label>
-                            </li>
-                            <li>
-                                <label class="checkbox"><input type="checkbox" data-type="group" />__MSG__GROUPS__</label>
-                            </li>
-                            <li>
-                                <label class="checkbox"><input type="checkbox" data-type="content" />__MSG__CONTENT__</label>
-                            </li>
-                            <li>
-                                <label class="checkbox"><input type="checkbox" data-type="folder" />__MSG__FOLDERS__</label>
-                            </li>
-                            <li>
-                                <label class="checkbox"><input type="checkbox" data-type="discussion" />__MSG__DISCUSSIONS__</label>
-                            </li>
-                        </ul>
+            <div class="oae-main-content">
+                <!-- EVERYWHERE / MY STUFF ONLY -->
+                <div id="search-scope" class="oae-switchtab-container">
+                    <div class="oae-switchtab-control">
+                        <button class="oae-switchtab-switch" data-switchtab-id="all" data-switchtab-path="/search/all">
+                            <span class="oae-switchtab-switch-icon fa fa-globe"></span>
+                            <span class="oae-switchtab-switch-label">__MSG__EVERYWHERE__</span>
+                        </button>
+                        <button class="oae-switchtab-switch" data-switchtab-id="my" data-switchtab-path="/search/my">
+                            <span class="oae-switchtab-switch-icon fa fa-lock"></span>
```

Comment:
I'd still be in favour of using the "Home" icon instead of the "Lock" icon, as it might otherwise communicate that "My stuff" will be limited to private items and using the "Home" icon connects it better to "My Personal Space". The "Globe" icon for "Search everywhere" is not ideal in that regard either, but we don't really have a better one.

+1 to adding color to the element icon instead of using the check icon.